python/segyio/tools.py

```diff
@@ -403,3 +407,429 @@ def resample(f, rate = None, delay = None, micro = False,
     f._samples = (np.arange(len(f.samples)) * rate) + t0
 
     return f
+
+
+def from_array(filename, data, iline=189,
+                               xline=193,
+                               sample_format=SegySampleFormat.IBM_FLOAT_4_BYTE,
+                               dt=4000,
+                               delay_recording_time=0):
```

Comment:
Personally, I think `delay_recording_time` and also to some extent `samle_format` is quite the mouthful. I guess `sample_format` is ok, but what do you think of the (SU-derived) `delrt` for delay-recording-time?

Reply:
I agree on using the `deltr`. What do you think of using just `format`. Although it redefines the built-in `format`, it should not cause any confusion due to the different nature of the two!?